### PR TITLE
[sr] Serbian Language translation

### PR DESCRIPTION
**Pull request by nebojsajsimic**

### CODE REVIEW

File: src/Language/sr/Auth.php [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    'suggestPasswordPwned'      => '{0} nikada ne treba biti korišćen za lozinku. Ako to koristite bilo gde promenite lozinku odmah.',
+    'errorPasswordEmpty'        => 'Lozinka je obavezna.',
+    'errorPasswordTooLongBytes' => 'Lozinka ne može preći {param} bytova dužine.',
+    'passwordChangeSuccess'     => 'Lozinka je uspešno promenjena',
+    'userDoesNotExist'          => 'Lozinka nije promenjena. Korisnični nalog ne postoji',
+    'resetTokenExpired'         => 'Žao nam je ali token je istekao.',
+
+    // Email Globals
+    'emailInfo'      => 'Neke informacije o osobi:',
+    'emailIpAddress' => 'IP Address:',
```
Comment (datamweb) [inline src/Language/sr/Auth.php:71]: How about?

File: src/Language/sr/Auth.php [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    'needAccount'        => 'Potreban Vam je nalog?',
+    'rememberMe'         => 'Zapmti me?',
+    'forgotPassword'     => 'Zaboravljena lozinka?',
+    'useMagicLink'       => 'Koristi pristupni link',
+    'magicLinkSubject'   => 'Vaš pristupni link',
+    'magicTokenNotFound' => 'Nije moguća verifikacija linka.',
+    'magicLinkExpired'   => 'Žao nam je, link je istekao.',
+    'checkYourEmail'     => 'Proverite Vaš email!',
+    'magicLinkDetails'   => 'Upravo smo Vam poslali pristupni link. Pristupni link će biti validan još samo {0} minuta.',
+    'successLogout'      => 'You have successfully logged out.',
```
Comment (datamweb) [inline src/Language/sr/Auth.php:50]: How about?

File: tests/Language/AbstractTranslationTestCase.php [hunk 3]
```diff
@@ -70,6 +70,7 @@ abstract class AbstractTranslationTestCase extends TestCase
         //        RussianTranslationTest::class            => 'ru',
         //        SinhalaTranslationTest::class            => 'si',
         SlovakTranslationTest::class  => 'sk',
+        SerbianTranslationTest::class  => 'sr',
```
Comment (datamweb) [inline tests/Language/AbstractTranslationTestCase.php:73]:
```suggestion
        SerbianTranslationTest::class => 'sr',
```

**Review by datamweb — CHANGES_REQUESTED** [review]

**Review by datamweb — APPROVED** [review]
@nebojsajsimic Thank you for PRs.
Welcome to The Shield.

**Review by kenjis — APPROVED** [review]

**Pull request merged**